src/Process/Builder/SubscriberBuilder.php

```diff
@@ -67,57 +66,29 @@
  */
 final class SubscriberBuilder
 {
-    /**
```

Comment:
I would move this decision to a separate discussion / issue since we didn't agree on this topic (AFAIK).

Even with constructor type inference support from PHPStan and now PHPStorm we should agree on this topic and, probably, if it makes sense, fix it once and for all by 1 commit (or add a rule to always have typehinst, depending on a decision).

Reply:
you mean just for the `@var` or something else?

Reply:
yes, sorry, just for the property PHPDoc typehints:

```php
/**
 * @var MetricsCalculator
 */
```